compute_sdk/tests/integration/test_web_client_exceptions.py

```diff
@@ -26,11 +26,11 @@ def client():
 
 @pytest.mark.parametrize("http_status", [400, 500])
 def test_reason_parsed_as_part_of_error(client, http_status):
```

Comment:
Should change the title of the text to reflect 'message' is being parsed, not 'reason'.

Also, the GlobusAPIError refactoring uses 'detail', 'title', and 'message' fields to populate its fields.  (message uses all 3, but errors uses only detail and title).  Haven't spent too much time figuring out what the intended use is, but using message should be ok for now.

Reply:
Good call on the function name.

What I can glean from the recent changes is that if we want to directly access the Globus API response message, we should use `GlobusAPIError.text`.